entities/src/api_req_params.rs

```diff
@@ -150,6 +181,17 @@ pub struct GetAssetsByOwner {
     pub cursor: Option<String>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
```

Comment:
redundant fields prevent de/serialization. is it expected behaviour?

Reply:
Yes, it's a common practise in working with client request. We use it widely for almost all requests

Reply:
Ok then